Packs/SekoiaXDR/Integrations/SekoiaXDR/SekoiaXDR_test.py

```diff
@@ -0,0 +1,658 @@
+    }
+    result = SekoiaXDR.add_keys_asset_command(client=client, args=args)
+
+    assert result.outputs["name"] == args["name"]
+    assert result.outputs["value"] == args["value"]
+
+
+def test_remove_keys_asset(client, requests_mock):
+    requests_mock.delete(
+        MOCK_URL
+        + "/v1/asset-management/assets/015ea33b-a7a2-4e34-8beb-0197a93a1011/keys/8007222c-f135-4f5f",
+        json={},
+    )
+
+    args = {
+        "asset_uuid": "015ea33b-a7a2-4e34-8beb-0197a93a1011",
+        "key_uuid": "8007222c-f135-4f5f",
+    }
+    result = SekoiaXDR.remove_key_asset_command(client=client, args=args)
+
+    assert not result.outputs
+
+
+def test_add_attr_asset(client, requests_mock):
+    mock_response = util_load_json("test_data/SekoiaXDR_post_asset_attr.json")
+    requests_mock.post(
+        MOCK_URL
+        + "/v1/asset-management/assets/015ea33b-a7a2-4e34-8beb-0197a93a1011/attr?name=attr_test_4&value=value4",
+        json=mock_response,
+    )
+
+    args = {
+        "asset_uuid": "015ea33b-a7a2-4e34-8beb-0197a93a1011",
+        "name": "attr_test_4",
+        "value": "value4",
+    }
+    result = SekoiaXDR.add_attributes_asset_command(client=client, args=args)
+
+    assert result.outputs["name"] == args["name"]
+    assert result.outputs["value"] == args["value"]
+
+
+def test_remove_attr_asset(client, requests_mock):
+    requests_mock.delete(
+        MOCK_URL
+        + "/v1/asset-management/assets/015ea33b-a7a2-4e34-8beb-0197a93a1011/attr/8007222c-f135-4f5f",
+        json={},
+    )
+
+    args = {
+        "asset_uuid": "015ea33b-a7a2-4e34-8beb-0197a93a1011",
+        "attribute_uuid": "8007222c-f135-4f5f",
+    }
+    result = SekoiaXDR.remove_attribute_asset_command(client=client, args=args)
+
+    assert not result.outputs
+
+
+def test_get_user(client, requests_mock):
+    mock_response = util_load_json("test_data/SekoiaXDR_get_user.json")
+    requests_mock.get(MOCK_URL + "/v1/users/7114c307-d86c-4c55", json=mock_response)
+
+    args = {"user_uuid": "7114c307-d86c-4c55"}
+    result = SekoiaXDR.get_user_command(client=client, args=args)
+
+    assert result.outputs["firstname"] == "Joe"
+    assert result.outputs["lastname"] == "done"
+
+
+def test_get_mapping_fields():
+    expected_mapping = {
+        "Sekoia XDR": {
+            "short_id": "The ID of the alert to edit",
+            "status": "The name of the status.",
+            "xsoar_id": "The XSOAR incident",
+        }
+    }
+    schema_result = SekoiaXDR.get_mapping_fields_command()
+
+    assert schema_result.extract_mapping() == expected_mapping
+
+
+def test_update_remote_system(client, requests_mock):
+    mock_response = util_load_json("test_data/SekoiaXDR_get_alerts.json")
+    requests_mock.get(MOCK_URL + "/v1/sic/alerts", json=mock_response)
+
+    args = {"lastUpdate": "2023-06-28T13:21:45"}
+    results = SekoiaXDR.get_modified_remote_data_command(client, args)
+
+    assert len(results.modified_incident_ids) == 2
```

Comment:
Which command do you want to test here? update_remote_system or get_modified_remote_data?